ffi/build.rs

```diff
@@ -0,0 +1,101 @@
+#[cfg(not(target_os = "windows"))]
+use other::main_stub;
+#[cfg(target_os = "windows")]
+use win::main_stub;
+
+fn main() {
+    main_stub();
+}
+
+#[cfg(target_os = "windows")]
+mod win {
+    extern crate embed_resource;
+
+    use std::env;
+    use std::fs::File;
+    use std::io::Write;
+
+    fn generate_version_rc() -> String {
+        let output_name = "DevolutionsIronRdp";
+        let filename = format!("{}.dll", output_name);
+        let company_name = "Devolutions Inc.";
+        let legal_copyright = format!("Copyright 2019-2022 {}", company_name);
```

Comment:
Copyright date needs updating (and probably wherever this was copied from)

Reply:
Thank you, both are updated, [picky.rs updated here](https://github.com/Devolutions/picky-rs/pull/259)